server/boot/user.js

```diff
@@ -631,4 +645,51 @@ module.exports = function(app) {
       return res.render('account/forgot');
     });
   }
+
+  function getReportUserProfile(req, res) {
+    const username = req.params.username.toLowerCase();
+    return res.render('account/report-profile', {
+      title: 'Report User',
+      username
+    });
+  }
+
+  function postReportUserProfile(req, res) {
+    const username = req.params.username.toLowerCase();
```

Comment:
req.params.username could be `undefined` causing this line to crash. 

Also, this line and the two below can be done declaratively using object destructuring.